### DISCUSSION TITLE
TASMOTA ALEXA HUE BRIDGE

### DISCUSSION THREAD

**Discussion by fpgiovanini** [arendst/Tasmota#22576] [Support request]
I configured tasmota device with 2 relays and 1 lamp (rgb).
I configured tasmota (others) with 3 friendly names and HUE BRIDGE
when I put Alexa (v3) device detection,Instead of 3 devices Alexa found only 1.
When deleting this 1 device, a new device appears and so on, 
until all 3 devices are deleted.
but if configure others to belkin wemos, all 3 devices are found but 
color control dont work.
Are the any solution for that?

**Comment by Jason2866** [comment]
Please use latest development builds when HUE Emulation is used. There was a change in Amazon API.
Needed changes for this are done after the last release of Tasmota

**Reply by fpgiovanini** [reply]
Thanks Jason2866. I´´try.
best regards
